docs/develop/dotnet/versioning.mdx

```diff
@@ -108,6 +108,90 @@ Implementing patching involves three steps:
 2. Remove the old code and apply [DeprecatePatch](https://dotnet.temporal.io/api/Temporalio.Workflows.Workflow.html#Temporalio_Workflows_Workflow_DeprecatePatch_System_String_).
 3. Once you're confident that all old Workflows have finished executing, remove `DeprecatePatch`.
 
+#### Overview
+
+The following sample shows how the `patched()` function behaves, providing explanations at each stage of the patching flow:
```

Comment:
Can we add a line about defining the version conditions in descending order?

Reply:
Something like 
> The best practice is to write your version conditionals in descending order so that newly executed workflows will set their marker as the latest version.

The default value for the version is `""` (empty string)